### PR TITLE
Add basic info about Certificate Manager

### PR DESCRIPTION
**Pull request by artemnikitin**
Comments and remarks are welcome because I'm not a native English speaker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1445,6 +1446,19 @@ SES
 
 -       🔸**Internet Access:** SES SMTP endpoints are on the Internet and will not be accessible from a location without Internet access (e.g. a private subnet without NAT gateway route in the routing table). In such a case, set up an SMTP relay instance in a subnet with Internet access and configure your application to send emails to this SMTP relay instance rather than SES. The relay should have a [forwarding rule to send all emails to SES](http://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-smtp-existing-server.html)). ❗If you are using a proxy instead of a NAT, confirm that your proxy service supports SMTP.
 
+Certificate Manager
+-------------------
+
+### Certificate Manager Basics
+-	📒 [Homepage](https://aws.amazon.com/certificate-manager/) ∙ [User guide](http://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html) ∙ [FAQ](https://aws.amazon.com/certificate-manager/faqs/) ∙ [Pricing](https://aws.amazon.com/certificate-manager/pricing/)
+- **Certificate Manager** is an AWS service for managing SSL/TLS certificates for AWS services.
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking phrasing suggestion: "Use the **Certificate Manager** to manage SSL/TLS certificates in other AWS services."

File: README.md [hunk 2]
```diff
@@ -1445,6 +1446,19 @@ SES
...
+- Supports import of already owned certificates and issuing new certificates.
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "Supports importing existing certificates as well as issuing new ones."

File: README.md [hunk 3]
```diff
@@ -1445,6 +1446,19 @@ SES
...
+
+### Certificate Manager Alternatives and Lock-in
+- Service allows import of existed certificates without limitations, but certificates issued via this service can't be used outside of AWS.
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "Certificates issued by the **Certificate Manager** can't be used outside of the services that support it. Imported certificates, however, can still be used elsewhere.

Comment (artemnikitin) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (artemnikitin) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (artemnikitin) [inline README.md]: Done

**Comment by artemnikitin** [comment]
> Definitely not required for this PR (I will file a ticket or just make the additions myself) but in the future it would be nice to have the release date of the Certificate Manager in the release history table, as well as listing the services that integrate with the Certificate Manager here.

If you mean table in **AWS Product Maturity and Releases**, then info is already there.

**Comment by ThanosBaskous** [comment]
Thanks @artemnikitin! You're right about about Certificate Manager already being in the "AWS Product Maturity and Releases" table - my mistake!

I'm assigning to @jlevy for second stage review.

**Comment by lynnlangit** [comment]
We should probably add that AWS Certificate Manager only works with Elastic Load Balancer and Cloud Front.  Also that AWS IoT has a its own process for creating and managing certificates intended to be used with IoT devices.  The latter servers as authorization and authentication (via IoT policies) for specific IoT devices to send device messages via MQTT to a particular AWS IoT Device Gateway (endpoint).

**Comment by artemnikitin** [comment]
@lynnlangit I planned to add this info a little bit later on this week into Tips and Gotchas

**Comment by jlevy** [comment]
👍 LGTM. Thanks @artemnikitin @lynnlangit !

**Comment by jlevy** [comment]
@artemnikitin adding that issue and giving to you, so we track it. Thanks again.

**Comment by artemnikitin** [comment]
@jlevy ok, thanks

**Pull request merged**